x-pack/plugins/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -25,37 +40,29 @@ const checkValueAgainstComparatorMap: {
   [Comparator.LT_OR_EQ]: (a: number, b: number) => a <= b,
 };
 
-export const createLogThresholdExecutor = (alertUUID: string, libs: InfraBackendLibs) =>
+export const createLogThresholdExecutor = (alertId: string, libs: InfraBackendLibs) =>
   async function ({ services, params }: AlertExecutorOptions) {
-    const { count, criteria } = params as LogDocumentCountAlertParams;
     const { alertInstanceFactory, savedObjectsClient, callCluster } = services;
     const { sources } = libs;
+    const castParams = params as LogDocumentCountAlertParams;
```

Comment:
I'm seeing that this has just been like this before, but maybe we could note down a follow-up to run this through a decoder instead of blindly casting it? (no power to the `NullPointerException` :wink:)